builtin_interfaces/Time stamp

uint16[] id_list
float64[] position_deg